interpreter/cling/lib/Utils/ASTImportSource.cpp

```diff
@@ -0,0 +1,189 @@
+#include "cling/Utils/ASTImportSource.h"
+
+using namespace clang;
+
+namespace cling {
+  namespace utils {
+    ASTImportSource::ASTImportSource(cling::Interpreter *parent_interpreter,
+                                     cling::Interpreter *child_interpreter) :
+      m_parent_Interp(parent_interpreter), m_child_Interp(child_interpreter) {
+    }
+
+    void ASTImportSource::ImportDecl(Decl *declToImport,
+                                     ASTImporter &importer,
+                                     DeclarationName &childDeclName,
+                                     DeclarationName &parentDeclName,
+                                     const DeclContext *childCurrentDeclContext) {
+
+      Decl *importedDecl = nullptr;
+      /* Don't do the import if we have a Function Template.
```

Comment:
We generally prefer C++ comments, throughout, except for ROOT's doxygen class doc. Can you convert all comments to `//`?